sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -403,6 +408,24 @@ public static void startFromLibrary(Context context, Class<? extends AppCenterSe
         getInstance().startInstanceFromLibrary(context, services);
     }
 
+    /**
+     * Allow or disallow network requests.
```

Comment:
Can we add more details here? I'd expect to see explanation about policies and differences with enabled flag